barman/retention_policies.py

```diff
@@ -126,6 +126,29 @@ def to_json(self):
         """
         return "%s %s %s" % (self.mode, self.value, self.unit)
 
+    def propagate_retention_status_to_children(self, backup_info, report, ret_status):
```

Comment:
```suggestion
    def _propagate_retention_status_to_children(self, backup_info, report, ret_status):
```

I think this should be an "internal" method as at a first glance we won't be calling this from the outside.